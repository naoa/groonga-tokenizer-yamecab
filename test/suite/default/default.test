register tokenizers/yamecab

load --table @yamecab
[
{"_key": "TokenYaMecab", "@updates":true}
]
tokenize TokenYaMecab "今日は、雨です"
