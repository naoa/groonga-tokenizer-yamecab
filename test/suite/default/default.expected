plugin_register tokenizers/yamecab
[[0,0.0,0.0],true]
tokenize TokenYaMecab "今日は、雨です"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false
    },
    {
      "value": "雨",
      "position": 3,
      "force_prefix": false
    }
  ]
]
